test/unit/webhook_template_test.rb

```diff
@@ -0,0 +1,19 @@
+
+require File.join(File.dirname(__FILE__), 'test_helper')
+
+describe HammerCLIForemanWebhooks::WebhookTemplate do
```

Comment:
If we have `foreman_api.json` available, then it's better to use functional tests.